src/utils/vacations-utils.ts

```diff
@@ -76,22 +76,30 @@ export function formatVacationDaysPayload(vacationDays: VacationDays): VacationD
  */
 export const validateVacationRequestDays = (
   requestedDays: number,
-  unspentDays: number
+  unspentDays: number,
+  options?: { isAdmin?: boolean }
```

Comment:
This seems a little strange to add this as options and then an object containing isAdmin, why not just pass the isAdminBoolean- would be clearer.